fe/fe-core/src/main/java/org/apache/doris/nereids/NereidsPlanner.java

```diff
@@ -228,6 +230,44 @@ public Plan planWithLock(LogicalPlan plan, PhysicalProperties requireProperties,
         }
     }
 
+    /**
```

Comment:
1. better to add this into pr desc
2. double-check the basic benchmark such as tpcds 1T olap/external table performance, after removing the config.